### ISSUE TITLE
Reference to file tailwindcss/resolveConfig

### ISSUE THREAD

**Issue by Damien-Lev** [aanckar/react-pdf-tailwind#19]
Hello, i got this error : 
![image](https://github.com/aanckar/react-pdf-tailwind/assets/42603404/6b0b6d37-5647-4d9c-a4f7-6bbb3e6b4ab9)

i'm using symfony ux-react, with webpack-encore. 

I'm maybe missing some configuration somewhere :'( 

Here are some files that may be interesting to help me:

tailwind.config.js
![image](https://github.com/aanckar/react-pdf-tailwind/assets/42603404/df334a31-2630-4fef-915a-362e07102937)

postcss.config.js
![image](https://github.com/aanckar/react-pdf-tailwind/assets/42603404/4c45fc46-2efa-41d6-b224-2b877bf63cce)

webpack.config.js
![image](https://github.com/aanckar/react-pdf-tailwind/assets/42603404/4ec5e77e-b227-4fb8-b584-ffc6f428654e)

**Comment by msichterman** [comment]
I am also getting this same error

**Comment by msichterman** [comment]
@Damien-Lev adding this to my webpack config fixes the issue
```
resolve: {
    extensions: ['.css', '.js', '.jsx', '.tsx', '.ts', '.cjs'],
    alias: {
      'tailwindcss/resolveConfig': 'tailwindcss/resolveConfig.js',
    },
  },
```

**Comment by EnriqueWolf** [comment]
The issue is also with Typescript. I added a path on compilerOptions in tsconfig.json but didn't work.
`"paths": {
    "tailwindcss/resolveConfig": ["node_modules/tailwindcss/resolveConfig.js"]
  }`

![2023-11-01_16h10_54](https://github.com/aanckar/react-pdf-tailwind/assets/148761639/d083651d-3401-4cd5-b6e0-1fb413f2681d)

**Comment by utopyin** [comment]
same issue on typescript!

**Comment by aanckar** [comment]
This is now hopefully fixed in [2.2.0](https://github.com/aanckar/react-pdf-tailwind/releases/tag/v2.2.0)
